core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3752,7 +3760,7 @@ public Object getDynamic(String token) {
     public synchronized void doConfigSubmit( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException, FormException {
         BulkChange bc = new BulkChange(this);
         try {
-            checkPermission(ADMINISTER);
+            checkPermission(MANAGE);
```

Comment:
Taking how the code below is implemented, it is likely to expose some Admin-only configuration sections to users with `Manage` permissions. Hiding controls from frontend is fine, but IIUC the manually prepared from submission would pass here.